corporate/views/billing_page.py

```diff
@@ -62,7 +62,10 @@ def payment_method_string(stripe_customer: stripe.Customer) -> str:
 
 
 @zulip_login_required
-def billing_home(request: HttpRequest) -> HttpResponse:
+@has_request_variables
+def billing_home(
+    request: HttpRequest, onboarding: Optional[str] = REQ(default=None)
```

Comment:
Can't we use check_bool here? The value of onboarding if provided would be `"true"`